src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add readMe.txt file, where described how to run test framework.

Reply:
Done